j2v/generation/processor.py

```diff
@@ -71,3 +77,19 @@ def post_process(self):
 
     def process_single_dict(self, python_dict):
         self.generator.collect_all_paths(current_dict=python_dict)
+
+    def _is_truthy(self, candidate_value):
```

Comment:
It's only about translating the input value when a program is executed as main.py. If we use this class from any other python class, we never need to translate python True or False. That saying, let's move invocation of _is_truthy to main.py. The function definition can be moved to main.py or helpers.

Reply:
Moved is_truthy to helpers and called it from main.py